lib/perl/Genome/Sample/Command/Import/Base.pm

```diff
@@ -44,10 +44,13 @@ class Genome::Sample::Command::Import::Base {
         # source
         _individual => { is => 'Genome::Individual', is_optional => 1, },
         _individual_name => { is => 'Text', },
+        _individual_attributes => { is => 'Hash', },
```

Comment:
I think we more commonly use `HASH`, since that matches `ref ($cmd->_individual_attributes)`.